solution/ui/regulations/.eslintrc.json

```diff
@@ -16,6 +16,7 @@
     "plugins": ["vue"],
     "rules": {
         "eqeqeq": "off",
+        "import/no-unresolved": "off",
```

Comment:
our vite configs need some tweaking in order for our (my) IDE to recognize aliased imports.  Until then, I'm turning this rule off.